<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>substractToolPage</class>
 <widget class="QWidget" name="substractToolPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>507</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>507</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pbReturnMainPage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSubstractTool">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Substract</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbBackground">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Background</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbObject">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Object</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtBackground">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>441</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtObject">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>441</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbPrbfgb">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Checked foreground/ Unchecked background
Indicate the inital mask in GrabCut algorithm</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbThreshold">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
   <property name="value">
    <number>25</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>206</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Threshold</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbSubstractmode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Checked substract object with one channel in HSV
Uncheked substract object with images in gray scale</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbHsvchannel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>2</number>
   </property>
   <property name="value">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>206</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>HSV channel. H=0, S=1, V=2</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbWindowratio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Window ratio when show the image</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbGaussianiterations">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Gaussian iteration to clean noise in both images</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbMaskDestiny">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Output</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMaskDestiny">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>441</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtOutputProcess">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>328</y>
     <width>541</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbNewObject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>120</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>New Object</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbShowOutput">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ShowOutput</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
